paimon-web-server/src/main/java/org/apache/paimon/web/server/controller/CatalogController.java

```diff
@@ -99,14 +70,14 @@ public R<List<CatalogInfo>> getCatalog() {
     /**
      * Removes a catalog with given catalog name.
      *
-     * @param catalogName The catalog name.
+     * @param catalogId The catalog id.
      * @return A response indicating the success or failure of the operation.
      */
-    @DeleteMapping("/remove/{catalogName}")
-    public R<Void> removeCatalog(@PathVariable String catalogName) {
-        QueryWrapper<CatalogInfo> queryWrapper = new QueryWrapper<>();
-        queryWrapper.eq("catalog_name", catalogName);
-        return catalogService.remove(queryWrapper)
+    @DeleteMapping("/remove/{catalogId}")
+    public R<Void> removeCatalog(@PathVariable String catalogId) {
```

Comment:
Could this support remove by catalog name?

Reply:
Why use catalog name? Wouldn't it be better to use catalog id?

Reply:
@hunter-cloud09, does the `removeCatalog` only support by id? IMO, the operation that removes catalog could support by catalog id or name.

Reply:
Do you mean to add an api?

Reply:
@hunter-cloud09, I mean that the parameter type could be `CatalogDTO`.

Reply:
> @hunter-cloud09, I mean that the parameter type could be `CatalogDTO`.

get it!